### ISSUE TITLE
Software Freedom Conservancy logo img tag has an empty alt attribute

### ISSUE THREAD

**Issue by pabs3** [ShaneCurcuru/chooseafoundation#18]
The Software Freedom Conservancy logo img tag has an empty alt attribute, while other organisations have an alt attribute with the text "Someorg logo".

**Comment by ShaneCurcuru** [comment]
Where, specifically, are you seeing this?  Viewing source, I have the alt text just fine:

img src="http://chooseafoundation.com/assets/images/conservancy.png" alt="Conservancy tree logo"

**Comment by pabs3** [comment]
I'm seeing it on the front page:

```
$ curl -s http://chooseafoundation.com/ | grep -i -B2 -A3 conservancy.png
            <img src=
              
                "http://chooseafoundation.com/assets/images/conservancy.png"
              
            alt=""
            style="max-height: 120px; max-width: 300px; object-fit: contain;">
```

Looks like the Conservancy definition is missing an alt tag:

```
$ sed -n /eclipse.png/,/alt:/p index.html 
  - image_path: assets/images/eclipse.png
    alt: "Eclipse Logo"
$ sed -n /conservancy.png/,/alt:/p index.html 
  - image_path: /assets/images/conservancy.png
    title: "Software freedom is most important for us."
    excerpt: "The Software Freedom Conservancy is a charity that helps promote, improve, develop, and defend Free, Libre, and Open Source Software (FLOSS) projects."
    url: "/foundations/conservancy"
    btn_label: "Learn about Conservancy"
    btn_class: "btn--inverse"

feature_row2:
  - image_path: assets/images/eclipse.png
    alt: "Eclipse Logo"
```

-- 
bye,
pabs

https://bonedaddy.net/pabs3/

**Comment by ShaneCurcuru** [comment]
Fixed in 98aabbbdd47c7a9fa818adf0516edbee6fafd490
